ttmar_starter.html

```diff
@@ -60,7 +59,42 @@
 
 
   <script>
+var firstEvent = document.querySelector("#first_paragraph")
+      firstEvent.addEventListener("click", function(){
+      document.body.style.background = "#ccc"
+})
+
+var secondEvent = document.querySelector("#second_paragraph")
+    secondEvent.addEventListener("click", function(){
+    document.querySelector(".second_paragraph").style.display = "block"
+    })
+var thirdEvent = document.querySelector("#first_chorus")
+    thirdEvent.addEventListener("click", function(evt){
+      evt.preventDefault();
+      document.querySelector(".first_chorus").style.display = "block"
+    })
+var fourthEvent = document.querySelector("#third_paragraph")
+    fourthEvent.addEventListener("click", function(evt){
+      evt.preventDefault();
+      var elements = document.querySelectorAll(".third_paragraph")
+      for (var i = 0; i < elements.length; i++){
+        elements[i].style.display = "block"
+  }
+})
+var fifthEvent = document.querySelector("#fourth_paragraph")
+    fifthEvent.addEventListener("click", function(evt){
+      evt.preventDefault();
+      var elements = document.querySelectorAll(".fourth_paragraph")
+      for (var i = 0; i < elements.length; i++){
+        elements[i].style.display = "block"
+    }
+    })
+
+
 
   </script>
 </body>
 </html>
+
+<!-- 1) How are we supposed to test this? The page shows up blank when opened in the browser,
+ and the javascript blocks can't be entered into the console separately.  -->
```

Comment:
When I ran your version of this file I didn't have any problem getting things to run as they were supposed to. I'd be happy to take a closer look if you want to show me in person, but it looks ok from my end.